crates/forge/test_utils/src/runner.rs

```diff
@@ -99,6 +105,33 @@ impl<'a> TestCase {
 
         dir.child("src/lib.cairo").touch().unwrap();
 
+        let snforge_std_path = Utf8PathBuf::from_str("../../snforge_std")
+            .unwrap()
+            .canonicalize_utf8()
+            .unwrap()
+            .to_string()
+            .replace('\\', "/");
```

Comment:
Why are we replacing this?

Reply:
`\\` are used on windows, this unifies paths across systems

Reply:
A comment would be nice though